doc/content/guides/connecting-gateways/ciscoixm/_index.md

```diff
@@ -0,0 +1,348 @@
+
+Then save the configuration by executing
+
+```
+Gateway# copy running-config startup-config
+```
+
+### Packet Forwarder Configuration
+
+> ⚠️ Keep in mind that the pre-installed packet forwarder is not supported by Cisco for production purposes.
+
+To run the packet forwarder, we'll make use of the **container** that is running on the gateway at all times.
+
+```
+Gateway# request shell container-console
+```
+You will be requested to enter the System Password. By default this is **admin**.
+
+Create the directory to store the Packet Forwarder configuration:
+
+```bash
+bash-3.2# mkdir /etc/pktfwd
+```
+Copy the packet forwarder to **/etc/pktfwd**:
+
+```bash
+bash-3.2# cp /tools/pkt_forwarder /etc/pktfwd/pkt_forwarder
+```
+
+Cisco provides a list of configuration templates. To list the available templates:
+
+```bash
+bash-3.2# ls -l /tools/templates
+  total 136
+  -rwxr-xr-x    1 65534    65534        11323 Oct  8 13:30 config_loc_dual_antenna_8ch_full_diversity_EU868.json
+  -rwxr-xr-x    1 65534    65534        11248 Oct  8 13:30 config_loc_dual_antenna_8ch_full_diversity_JP920.json
+  -rwxr-xr-x    1 65534    65534        11323 Oct  8 13:30 config_loc_dual_antenna_8ch_partial_diversity_EU868.json
+  -rwxr-xr-x    1 65534    65534         7993 Oct  8 13:30 config_loc_single_antenna_16ch_EU868.json
+  -rwxr-xr-x    1 65534    65534         7080 Oct  8 13:30 config_loc_single_antenna_16ch_US915.json
+  -rwxr-xr-x    1 65534    65534        13519 Oct  8 13:30 config_loc_single_antenna_64ch_US915.json
+  -rwxr-xr-x    1 65534    65534        13635 Oct  8 13:30 config_loc_single_antenna_full_duplex_64ch_US915.json
+  -rwxr-xr-x    1 65534    65534        17478 Oct  8 13:30 config_test_dual_antenna_56ch_partial_diversity_EU868.json
+  -rwxr-xr-x    1 65534    65534        14148 Oct  8 13:30 config_test_single_antenna_64ch_64x1_EU868.json
+  -rwxr-xr-x    1 65534    65534        14148 Oct  8 13:30 config_test_single_antenna_64ch_8x8_EU868.json
+```
+
+Copy the configuration template **config_loc_single_antenna_16ch_EU868.json** as **config.json** to **/etc/pktfwd** 
+
+```bash
+bash-3.2# cp /tools/templates/config_loc_single_antenna_16ch_EU868.json /etc/pktfwd/config.json
+```
+
+> Note: choose the Choose the file applicable to your region.
+
+Edit the configuration using a text editor, such as **vi**:
+
+```bash
+bash-3.2# vi /etc/pktfwd/config.json
+```
+
+>Note: Press the `i` key on your keyboard to start insert mode. Once finished editing, press `ESC` and enter `:wq` to write the file and quit.
+
+Edit the **gateway_conf** parameters.
+
+1. **gateway_ID**: The **EUI** that you used for registration.
+2. **server_address**: The **Gateway Server Address** that you used for registration.
+3. **serv_port_up**: UDP upstream port of the Gateway Server, typically **1700**.
+4. **serv_port_down**: UDP downstream port of the Gateway Server, typically **1700**.
```

Comment:
This is what GCS does for us. Please see the NASys and Kerlink (i)Station docs

Reply:
what about the Tektelic Kona Micro IoT doc then, it has the same issue, that's why I did it this way.

Reply:
discussed in the comment above

Reply:
Discussed in the comment above.

Reply:
Done